cmd.go

```diff
@@ -487,8 +597,23 @@ func doOutput(args []string) {
 	})
 }
 
-func parseTestArguments(args []string) (packages, testFlags []string) {
-	// format: [-godebugwork] [-instrument pkgs...] [packages] [testFlags]
+func parseBuildArguments(args []string) ([]string, bool) {
+	if len(args) == 0 {
+		return []string{"."}, true
+	}
+	if len(args) == 1 && !strings.HasSuffix(args[0], ".go") {
+		return args, true
+	}
+	for _, a := range args {
+		if !strings.HasSuffix(a, ".go") {
+			logFatal("you can only build a set of files or a single package")
+		}
+	}
+	return args, false
+}
+
+func parseTestArguments(args []string) ([]string, []string) {
```

Comment:
I find the named return parameters to be helpful documentation here. Was there a reason you removed them?